src/pydap/parsers/dmr.py

```diff
@@ -80,12 +81,15 @@ def get_dtype(element):
     return dtype
 
 
-def get_attributes(element):
-    attributes = {}
+def get_attributes(element, attributes={}):
     attribute_elements = element.findall("Attribute")
+    numType = [item for item in dmr_atomic_types if item not in ["Byte", "Char"]]
     for attribute_element in attribute_elements:
         name = attribute_element.get("name")
         value = attribute_element.find("Value").text
+        _type = attribute_element.get("type")

```

Comment:
This bit comes in handy for the cases where Missing value or Fill_value have a Value that is of string type. To follow CF conventions, the type of Missing_value and Fill_value must of a numeric type.